README.md

```diff
@@ -64,7 +64,20 @@ click the edit button at the bottom right of the page.
 You'll need to be added as a contributor to this repository in order to
 authenticate with Netlify CMS.
 
-To run the documentation website locally:
+
+### Running the documentation website locally
+The project has a dependency on Ruby because it uses Jekyll. If you do not have Ruby installed, you will need to install it. We recommend using [RVM](https://rvm.io/rvm/install). If you don't have admin access to your machine, try these steps:
+
+```shell
+curl -sSL https://get.rvm.io | bash -s stable --ruby
+rvm install 2.7
```

Comment:
Why 2.7? I use 3.1.1 locally.

Reply:
@flacoman91 just tried installing it locally and had trouble with the eventmachine install step and we were able to get it going with 2.7. If you have 3.1.1 steps you're confident with, I'm happy to go with those (and it would close https://github.com/cfpb/design-system/issues/1385)

Reply:
I had ruby 3.0 and it kept dying on me.   The instructions in that ticket said that it would work for 3.0, but I could not get it to work..

I can get a few other testers here if @KatherynRizek or anyone else from data products team has some time to give these steps a try. Today was the first time i tried running it.